### ISSUE TITLE
navigateAction should support routeName, navParams

### ISSUE THREAD

**Issue by jwalton** [yahoo/fluxible-router#76]
It would be really slick if, in addition to supporting `{method, url}`, navigateAction would also support `{routeName, navParams}` like `NavLink` does.  (Also, bonus points if both of these could accept a `query`.)

Happy to provide a PR.  I have a very hacky coffee-script version already:  ;)

```
fluxibleRouter  = require 'fluxible-router'
ajax = require '../../lib/ajax'

module.exports = (context, {method, url, routeName, navParams, query}) ->
    if !url?
        routeStore = context.getStore('RouteStore')
        url = routeStore.makePath routeName, navParams
        if !url then throw new Error "Couldn't make URL!"
        if query? then url += "?" + ajax.encodeQuery query

    context.executeAction fluxibleRouter.navigateAction, {method, url}
```

Very handy when you want to navigate to a new URL from a button click or some other non-link action.

**Comment by umidbekkarimov** [comment]
+1, it's very annoying to manually compose url

**Comment by redonkulus** [comment]
@jwalton sounds like a good enhancements, please feel free to open a PR.

**Comment by opportunato** [comment]
Provided a solution with tests, also updated description.

**Comment by mridgway** [comment]
Fixed in #89

**Comment by jwalton** [comment]
Thanks for fixing this; was planning a PR, but I've been swamped with other
work.  :)
